docs/paper/dev/getting-started/paper-plugins.md

```diff
@@ -0,0 +1,103 @@
+---
+slug: /dev/getting-started/paper-plugins
+---
+
+# Paper Plugins
+
+Paper plugins allow developers to take advantage of more modern concepts introduced by mojang, such as datapacks, to 
+expand the field of what the Paper API is able to introduce.
+
+:::info Experimental
+
+This is experimental and may be subject to change.
+
+:::
+
+- [Bootstrapper](#bootstrapper)
+- [Loader](#loaders)
+- [Differences](#differences)
+
+## How do I use them?
+Similarly to Bukkit plugins, you have to introduce a ``paper-plugin.yml`` file into your jar resources folder.
+Here is an example configuration.
+```yml
+name: Paper-Test-Plugin
+version: ${version}
+main: io.papermc.testplugin.TestPlugin
+description: Paper Test Plugin
+author: PaperMC
+api-version: ${apiversion}
+load: STARTUP
+bootstrapper: io.papermc.testplugin.TestPluginBootstrap
+loader: io.papermc.testplugin.TestPluginLoader
+defaultPerm: FALSE
+permissions:
+dependencies:
+```
+
+## What is it used for?
+Paper plugins lay down the framework for some future api.
+
+
+## Bootstrapper
+Paper plugins are able to identify their own bootstrapper by implementing ``io.papermc.paper.plugin.bootstrap.PluginBootstrap`` and adding
+the class of your implementation to the bootstrapper field in the ``paper-plugin.yml``.
+```java
+public class TestPluginBootstrap implements PluginBootstrap {
+
+    @Override
+    public void bootstrap(@NotNull PluginProviderContext context) {
+        
+    }
+
+    @Override
+    public @NotNull JavaPlugin createPlugin(@NotNull PluginProviderContext context) {
+        return new TestPlugin("My custom parameter");
+    }
+    
+}
+```
+A Bootstrapper allow you to also override how your plugin is initiated, allowing you to pass values into your plugin constructor.
+Currently, bootstrappers do not offer much new api, and are highly experimental. This may be subject to change once more api is more introduced.
```

Comment:
```suggestion
Currently, bootstrappers do not offer much new API, and are highly experimental. This may be subject to change once more API is introduced.
```